<launch>
  <!-- No namespace here as we will share this description.
       Access with slash at the beginning -->
  <param name="robot_description"
    command="$(find xacro)/xacro --inorder $(find turtlebot3_description)/urdf/turtlebot3_burger.urdf.xacro" />


  <!-- BEGIN ROBOT 1-->
  <group ns="robot1">
    <param name="tf_prefix" value="robot1_tf" />
    <include file="$(find cs685)/launch/one_robot_model.launch" >
      <arg name="init_pose" value="-x -0.5 -y -1.0 -z 0 -Y 1.57" />
      <arg name="robot_name"  value="robot1" />
    </include>
  </group>

  <!-- BEGIN ROBOT 2-->
  <group ns="robot2">
    <param name="tf_prefix" value="robot2_tf" />
    <include file="$(find cs685)/launch/one_robot_model.launch" >
      <arg name="init_pose" value="-x -2.0 -y -0.5 -z 0 -Y 0.0" />
      <arg name="robot_name"  value="robot2" />
    </include>
  </group>

  <!-- BEGIN ROBOT 3-->
  <group ns="robot3">
    <param name="tf_prefix" value="robot3_tf" />
    <include file="$(find cs685)/launch/one_robot_model.launch" >
      <arg name="init_pose" value="-x 1.5 -y -0.5 -z 0 -Y 3.14" />
      <arg name="robot_name"  value="robot3" />
    </include>
  </group>

</launch>